lib/ssz_ex.ex

```diff
@@ -582,14 +677,22 @@ defmodule LambdaEthereumConsensus.SszEx do
   end
 
   # https://notes.ethereum.org/ruKvDXl6QOW3gnqVYb8ezA?view
-  defp sanitize_offset(offset, previous_offset, num_bytes, num_fixed_bytes) do
+  defp sanitize_offset(offset, previous_offset, _num_bytes, num_fixed_bytes)
```

Comment:
If we're always going to re-assign to the same variable, instead of sanityze I'd call this `check_offset_valid` and return `:ok` if that's the case.